src/screens/OtherProfile/index.js

```diff
@@ -370,11 +370,13 @@ const OtherProfile = () => {
     setInitLoading(false);
   };
 
-  const fetchOtherProfile = async () => {
+  const fetchOtherProfile = async (preventCache = false) => {
     const status = await netInfo.fetch();
     if (status.isConnected) {
       try {
-        handleOfflineMode();
+        if (!preventCache) {
+          handleOfflineMode();
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fetchOtherProfile` function now accepts a `preventCache` parameter. This seems to be used to control whether or not to handle offline mode. Make sure that all calls to this function have been updated to pass this parameter where necessary.
```diff
- const fetchOtherProfile = async () => {
+ const fetchOtherProfile = async (preventCache = false) => {
+   if (!preventCache) {
+     handleOfflineMode();
+   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->